apps/notifications/src/panel/templates/index.jsx

```diff
@@ -1,12 +1,14 @@
 /**
- * @module templates/index
+ * External dependencies
```

Comment:
Are the changes in this file related to this PR? I'm ok with keeping them, but I just want to make sure.

Reply:
Some changes are ESLint fixes, this one related to missing internal/external docblock comments. And I also removed the `@module` docblock pragma -- we don't use it anywhere else.